### PR TITLE
fix(importer): pass context during collaboration import

### PR DESCRIPTION
**Pull request by marstamm**

### CODE REVIEW

File: test/spec/import/BpmnTreeWalkerSpec.js [hunk 1]
```diff
@@ -138,13 +168,13 @@ function createWalker(listeners) {
...
-      listeners.element && listeners.element(element, parent);
+      return listeners.element && listeners.element(element, parent);
     },
     root: function(root) {
-      listeners.root && listeners.root(root);
+      return listeners.root && listeners.root(root);
     },
     error: function(message, context) {
-      listeners.error && listeners.error(message, context);
+      return listeners.error && listeners.error(message, context);
```
[the remark this answers is not in the document]
Comment (marstamm) [reply test/spec/import/BpmnTreeWalkerSpec.js:177]: To mimic the behavior of the Importer. We use the return values in the tree walker to establish the parent chain:
https://github.com/bpmn-io/bpmn-js/blob/master/lib/import/Importer.js#L52-L63
https://github.com/bpmn-io/bpmn-js/blob/master/lib/import/BpmnTreeWalker.js#L83

File: test/spec/import/BpmnTreeWalkerSpec.js [hunk 2]
```diff
@@ -50,6 +51,35 @@ describe('import - BpmnTreeWalker', function() {
...
+    return createModdle(collaboration).then(function(result) {
+
+      var definitions = result.rootElement;
+
+      // when
+      walker.handleDefinitions(definitions);
+
+      // then
+      expect(elementSpy.neverCalledWith(sinon.match.any, sinon.match.typeOf('undefined'))).to.be.true;
+      expect(errorSpy.notCalled).to.be.true;
```
Comment (nikku) [inline test/spec/import/BpmnTreeWalkerSpec.js:78]:
```suggestion
      expect(errorSpy).not.to.be.called;

```

Comment (marstamm) [reply test/spec/import/BpmnTreeWalkerSpec.js:78]: :+1: 

Will change it with the other tests in this file as well

**Comment by marstamm** [comment]
Fixes applied

**Review by nikku — APPROVED** [review]
:clap:

**Pull request merged**